### PR TITLE
Check license file for valid setup commands

### PR DESCRIPTION
**Pull request by WilliamBZA**

### CODE REVIEW

**Comment by WilliamBZA** [comment]
Tests pass @yvesgoeleven @HEskandari . Ready for review

**Review by yvesgoeleven — APPROVED** [review]

File: src/ServiceControl.LicenseManagement/LicenseManager.cs [hunk 1]
```diff
@@ -1,11 +1,11 @@
-﻿namespace ServiceControlInstaller.Engine.LicenseMgmt
+﻿namespace ServiceControl.LicenseManagement
 {
     using System;
     using Particular.Licensing;
 
     public class LicenseManager
     {
-        public static DetectedLicense FindLicense()
+        public static DetectedLicense FindLicense(bool includeTrial = true)
```
Comment (HEskandari) [inline src/ServiceControl.LicenseManagement/LicenseManager.cs:8]: I think I added this, but I don't think this is necessary. It should be:

```suggestion
        public static DetectedLicense FindLicense()
```

File: src/ServiceControl.LicenseManagement/ServiceControl.LicenseManagement.csproj [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net40</TargetFramework>
```
Comment (HEskandari) [inline src/ServiceControl.LicenseManagement/ServiceControl.LicenseManagement.csproj:4]: is this the right target framework?

**Review by HEskandari — COMMENTED** [review]
Reviewed. Two small comments.

File: src/ServiceControl.LicenseManagement/ServiceControl.LicenseManagement.csproj (hunk 2 above)
Comment (WilliamBZA) [reply src/ServiceControl.LicenseManagement/ServiceControl.LicenseManagement.csproj:4]: Yes, it's used by the installer project which is 40 so this has to match

**Pull request merged**